tests/cross/src/test/scala/tests/pc/CompletionMatchSuite.scala

```diff
@@ -200,21 +200,21 @@ class CompletionMatchSuite extends BaseCompletionSuite {
       |object Main {
       |  (null: AccessMode) match@@
       |}""".stripMargin,
-    """
-      |package example
-      |
-      |import java.nio.file.AccessMode
-      |import java.nio.file.AccessMode.READ
-      |import java.nio.file.AccessMode.WRITE
-      |import java.nio.file.AccessMode.EXECUTE
-      |
-      |object Main {
-      |  (null: AccessMode) match {
-      |\tcase READ => $0
-      |\tcase WRITE =>
-      |\tcase EXECUTE =>
-      |}
-      |}""".stripMargin,
+    s"""
```

Comment:
The change here makes sure that `\t` are translated to tabs. It doesn't work without `s` interpolator